linter/allowlists/default.txt

```diff
@@ -230,6 +231,7 @@ Usercentrics
 VR
 Valhalla
 Verband
+Verification
```

Comment:
Have these two words been complained by the linter? I'm wondering about that, because these are just common English words.

Reply:
Yes, I think it's because of the uppercase `V`. But I can check again locally.

Reply:
Yep:

```
$ ./linter/lint 

Contao Package metadata linter
==============================

Validating package names
 510/510 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

Validating metadata files
 412/763 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░]  53%
                                                                                            
 [ERROR] [Package: inspiredminds/contao-address-verification; Language: de]: Property       
         "title" does not pass the spell checker. Either update the allow list or fix the   
         spelling :) Errors: Verification                                                   
                                                                                            

                                                                                            
 [ERROR] [Package: inspiredminds/contao-address-verification; Language: de]: The YAML file  
         contains invalid data.                                                             
                                                                                            

 763/763 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

Validating composer.json files
 3/3 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%
```

Reply:
I moved it to the German allow list now.